token/cli/src/main.rs

```diff
@@ -4252,6 +4345,61 @@ fn app<'a, 'b>(
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateConfidentialTransferSettings.into())
+                .about("Update confidential transfer configuation for a token")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The address of the token mint to update confidential transfer configuration for")
+                )
+                .arg(
+                    Arg::with_name("approve_policy")
+                        .long("approve-policy")
+                        .value_name("APPROVE_POLICY")
+                        .takes_value(true)
+                        .index(2)
+                        .possible_values(&["auto", "manual"])
+                        .help(
+                            "Enable accounts to make confidential transfers. If \"auto\" \
+                            is selected, then accounts are automatically approved to make \
+                            confidential transfers. If \"manual\" is selected, then the \
+                            confidential transfer mint authority must approve each account \
+                            before it can make confidential transfers."
+                        )
+                )
+                .arg(
+                    Arg::with_name("auditor_pubkey")
+                        .long("auditor-pubkey")
+                        .value_name("AUDITOR_PUBKEY")
+                        .takes_value(true)
+                        .index(3)
+                        .help(
+                            "The auditor encryption public key. The corresponding private key for \
+                            this auditor public key can be used to decrypt all confidential \
+                            transfers involving tokens from this mint. Currently, the auditor \
+                            public key can only be specified as a direct *base64* encoding of \
+                            an ElGamal public key. More methods of specifying the auditor public \
+                            key will be supported in a future version. To disable auditability \
+                            feature for the token, use \"none\"."
+                        )
+                )
+                .arg(
+                    Arg::with_name("confidential_transfer_authority")
+                        .long("confidential-transfer-authority")
+                        .validator(is_valid_signer)
+                        .value_name("SIGNER")
+                        .takes_value(true)
+                        .help(
+                            "Specify the confidential transfer authority keypair. \
+                            Defaults to the client keypair address."
+                        )
+                )
```

Comment:
You've done the sign-only work already, so how about adding
```suggestion
                )
                .nonce_args(true)
                .offline_args(),
```

Reply:
Oh yes, we should add that as well!